@here/harp-mapview/lib/composing/MSAARenderPass.ts

```diff
@@ -187,7 +187,9 @@ export class MSAARenderPass extends Pass {
             // 6. Render the quad on top of the previous renders.
 
             // NOTE: three.js doesn't like undefined as renderTarget, but works with `null`
-            renderer.setRenderTarget(this.renderToScreen ? null! : writeBuffer);
+            renderer.setRenderTarget(
+                this.renderToScreen ? null : writeBuffer !== undefined ? writeBuffer : null
```

Comment:
You could instead define in LowResRenderPass.ts, MSAARenderPass.ts, Pass.ts
 writeBuffer: THREE.WebGLRenderTarget | null,
 readBuffer: THREE.WebGLRenderTarget | null

Reply:
done